feature-libs/product-configurator/common/components/configure-cart-entry/configure-cart-entry.component.ts

```diff
@@ -53,14 +57,21 @@ export class ConfigureCartEntryComponent {
       throw new Error('No entryNumber present in entry');
     }
 
-    return this.cartEntry.orderCode
-      ? this.commonConfigUtilsService.getComposedOwnerId(
-          this.cartEntry.orderCode,
-          entryNumber
-        )
+    const code = this.getCode();
+    return code
+      ? this.commonConfigUtilsService.getComposedOwnerId(code, entryNumber)
       : entryNumber.toString();
   }
 
+  protected getCode(): string | undefined {
+    if (this.cartEntry.quoteCode) {
+      return this.cartEntry.quoteCode;
+    } else if (this.cartEntry.orderCode) {
+      return this.cartEntry.orderCode;
+    }
+    return undefined;
```

Comment:
same as for OWNER_TYPE: is it desired that we return the quoteCode if both  quote  and order code are defined? If yes a test for this case would be good.

Reply:
Agree will improve that